app/services/item_servicer.rb

```diff
@@ -0,0 +1,15 @@
+class ItemServicer
```

Comment:
I'm curious if there's a reason you opted to name this class `Servicer` vs `Service`. I think you'll find `Service` is more commonly seen as the convention is Rails, but I'm interested in your thought process here.